expyfun/stimuli/tests/test_tracker.py

```diff
@@ -72,11 +74,28 @@ def test_tracker_ud():
     tr = TrackerUD(None, 3, 1, [1, 0.5], [1, 0.5], 10, np.inf, 1,
                    change_indices=[2])
     tr.respond(True)
-    tr = TrackerUD(None, 3, 1, [1, 0.5], [1, 0.5], np.inf, 10, 1,
-                   change_indices=[2], change_rule='trials')
-    for t in np.arange(2):  # run long enough to encounter change_indices
-        tr.respond(True)
-        tr.respond(False)
+
+    with warnings.catch_warnings(record=True) as w:
+        tr = TrackerUD(None, 1, 1, 0.75, 0.75, np.inf, 8, 1,
+                       x_min=0, x_max=2)
+        responses = [True, True, True, False, False, False, False, True, False]
+        for r in responses:  # run long enough to encounter change_indices
+            tr.respond(r)
+        assert_equal(len(w), 1)
+    assert(tr.check_valid(1))  # make sure checking validity is good
+    assert(not tr.check_valid(3))
```

Comment:
technically these should be `assert_true` but we'll probably change over to `py.test` soonish so it's okay